dissect/target/plugin.py

```diff
@@ -218,6 +218,34 @@ def __init_subclass__(cls, **kwargs):
     def __init__(self, target: Target):
         self.target = target
 
+    def is_compatible(self) -> bool:
+        """Perform a compatibility check with the target.
+
+        Perform a compatibility check with the target
+        without raising any exceptions and optionally in a
+        more perrformant manner
+
+        """
+        try:
+            self.check_compatible()
+        except UnsupportedPluginError:
+            return False
+
+        return True
+
+    @staticmethod
+    def is_findable() -> bool:
```

Comment:
Changed, but without _ because in other comments, there was the request to move the logic to helpers, so it now needs to be accessible from the outside.

Reply:
But if it's a public property one can never be sure the value hasn't been altered anywhere else in the code right? Python does not have read-only properties as far as I know.

Reply:
That's not possible, but such is life when using Python. It's ok I guess if someone explicitly changes this value for an instance at some point. Normally you use a single or double `_` to indicate there will be dragons when you do this.
Maybe specifying a `_findable` class variable and returning that in `findable()` will get the best of both worlds, configuring subclasses will be a simple line of code, the variable is indicated to be private and there is a public way of getting the value through the function.